<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pmd</class>
 <widget class="QDialog" name="pmd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="Speed_Initial2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>运行速度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Speed_Initial0">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Speed_Initial1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Adobe 黑体 Std R&quot;;</string>
   </property>
   <property name="text">
    <string>机台运行速度设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>初始速度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Spee_Set2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>初始速度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Acceleration2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>440</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Spee_Set1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>加速时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>加速时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>初始速度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Acceleration1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>运行速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>加速时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>运行速度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Acceleration0">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Spee_Set0">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ALL_CONFIRM">
   <property name="geometry">
    <rect>
     <x>1093</x>
     <y>690</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CHANGE">
   <property name="geometry">
    <rect>
     <x>973</x>
     <y>690</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>更改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CLOSE">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>690</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>30</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Adobe 黑体 Std R&quot;;</string>
   </property>
   <property name="text">
    <string>检测位置设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
